File: 1464-maximum-product-of-two-elements-in-an-array/1464-maximum-product-of-two-elements-in-an-array.c
int maxProduct(int* nums, int numsSize){
    int i,j;
    int max = (nums[0]-1) * (nums[1]-1);
    for(i = 0 ; i < numsSize - 1; ++i)
    {
        j = i + 1;
        for(; j < numsSize; ++j)
        {
            if((nums[i]-1) * (nums[j]-1) > max)
                max = (nums[i]-1) * (nums[j]-1);
        }
    }
    return max;
}

File: 1464-maximum-product-of-two-elements-in-an-array/1464-maximum-product-of-two-elements-in-an-array.java
class Solution {
    public int maxProduct(int[] nums) {
        int max = -1;
        for(int i = 0; i < nums.length-1; i++){
            for(int j = i+1; j < nums.length; j++){
                if ((nums[i] - 1) * (nums[j] - 1) > max)
                    max = (nums[i] - 1) * (nums[j] - 1);
            }
        }
        return max;
    }
}